distributed/tests/test_scheduler.py

```diff
@@ -1635,11 +1635,11 @@ async def test_idle_timeout(c, s, a, b):
 @gen_cluster(client=True, config={"distributed.scheduler.bandwidth": "100 GB"})
 async def test_bandwidth(c, s, a, b):
     start = s.bandwidth
-    x = c.submit(operator.mul, b"0", 1000000, workers=a.address)
+    x = c.submit(operator.mul, b"0", 1000001, workers=a.address)
```

Comment:
Why this change?

Reply:
When using `len` the `metrics["bandwidth"]["total"]` is reported to be `100000000000`, always.  When using `getsize` of the `metrics["bandwidth"]["total"]` at the end is `95012655208`.

Reply:
Understood.  I think I'm still confused on why we changed 1000000 to 1000001 here though.

Reply:
After a bit more digging I we need to increase here because we only update the bandwidth if the total data is larger than  1_000_000: https://github.com/dask/distributed/blob/7cdba1d7e17df7be005f4d4b8ca528065a3f83a7/distributed/worker.py#L2020-L2029

So to wrap this all up, when using `getsize` we are just over the line and we update bandwidth.  My guess is that we picked that value because we didn't want to update when passing small data such as heartbeats

Reply:
Ah ok, that makes sense.  Thanks for the explanation @quasiben !